src/App.js

```diff
@@ -10,7 +10,7 @@ import MainLayout from './components/layout/MainLayout/MainLayout';
 import Homepage from './components/views/Homepage/Homepage';
 import ProductList from './components/views/ProductList/ProductList';
 import ProductPage from './components/views/ProductPage/ProductPage';
-
+import BlogArticle from './components/views/BlogArticle/BlogArticle';
```

Comment:
Proponuje dać pustą linie po tym imporcie, aby nie zlewało się to razem z definicją komponentu